src/index.js

```diff
@@ -36,7 +36,7 @@ i18n
 const root = ReactDOM.createRoot(document.getElementById('app'));
 root.render(<App />);
 
-if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
+if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator && process.env.SERVICE_WORKER_DISABLED !== 'true') {
```

Comment:
in this case if you set `SERVICE_WORKER_DISABLED` to `true` in webpack.config.js, it will not disable it because it's parsed as boolean instead of a string
```suggestion
if (process.env.NODE_ENV === 'production' && process.env.SERVICE_WORKER_DISABLED !== 'true' && process.env.SERVICE_WORKER_DISABLED !== true && 'serviceWorker' in navigator) {
```

Reply:
Great catch, thank you! I
I've committed your suggestion.